SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -247,7 +247,7 @@ fi
 *   Mon Aug 07 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 0.6.0-2
 -   Bump release to rebuild with go 1.19.12
 
-*   Tue Jul 11 2023 Dallas Delaney <dadelan@microsoft.com> 0.6.0-1
+*   Tue Jul 28 2023 Dallas Delaney <dadelan@microsoft.com> 0.6.0-1
```

Comment:
nit - Is there a reason for this change?

Reply:
This is changing the date to be incorrect

Reply:
This was because I found an rpmspec --parse `error: %changelog not in descending chronological order` when running our toolkit script to update the cgmanifest for this package (the preceding changelog entry has a more recent date, when all entries should be strictly from newer to older). The commit was actually merged on Jul 28 https://github.com/microsoft/azurelinux/commit/c678e6f89038f8844be0bd1fced1716704baa057. I see what you are saying, it should be Friday. I'll make that update